src/fees/selectors.ts

```diff
@@ -9,12 +10,16 @@ export function getFeeInTokens(feeInWei: BigNumber.Value | null | undefined) {
 
 export const feeEstimatesSelector = (state: RootState) => state.fees.estimates
 
-export function getFeeEstimateDollars(feeType: FeeType | null, tokenAddress: string) {
+export function getFeeEstimateDollars(
+  feeType: FeeType | null,
+  tokenInfo: TokenBalance | undefined
+) {
   return (state: RootState) => {
-    if (!feeType) {
+    // TODO: Handle cases where address is null ex: Ethereum
```

Comment:
```suggestion
    // TODO(ACT-922): Handle cases where address is null ex: Ethereum
```
is [ACT-922](https://linear.app/valora/issue/ACT-922/ethereum-fee-estimation) when you envisioned this happening?

when you leave a TODO can you try to leave some indication of when or by whom it should get done?